lib/AnyEvent/RabbitMQ/Channel.pm

```diff
@@ -543,13 +543,14 @@ sub consume {
 
     my $consumer_cb = delete $args{on_consume} || sub {};
     my $cancel_cb   = delete $args{on_cancel} || sub {};
+    my $no_ack      = delete $args{no_ack} || 1;
```

Comment:
Does this want to be `//` rather than `||`? You otherwise cannot set `no_ack` to `0`.